### ISSUE TITLE
Add Voronoi Sites Matrix into node "Voronoi on Mesh"

### ISSUE THREAD

**Issue by satabol** [nortikin/sverchok#5181]
Sverchok master 1.3.

**What do you think?**

![image](https://github.com/user-attachments/assets/1aa99f13-556f-466a-ad85-ab954122442f)

This socket will apply transform source mesh on every site to simplify creation irregular object by Voronoi algorithm. Expected results:

![image](https://github.com/user-attachments/assets/a79dc009-ec89-4a0d-ada9-182940b292b6)

**Only Scales:**
![image](https://github.com/user-attachments/assets/274f7a28-f4a8-48c4-b8e9-f4bebdced310)

**Only Rotates:**
![image](https://github.com/user-attachments/assets/ada9904e-abc4-41e8-931d-6262f291c474)

**What do you think?**

**Comment by portnov** [comment]
Not sure I understand what it does. Transforms separated parts? What's difference with transforming them manually afterwards?

**Comment by satabol** [comment]
Hi.
> Transforms separated parts?
Algorithm create every part of whole source mesh. I suppose transform every source mesh on every site before bite source mesh on site. If transform parts after Voronoi then parts can overlap each other.

**Comment by satabol** [comment]
Example:
![image](https://github.com/user-attachments/assets/3874b19f-0da7-4bc0-a99f-cfa686f6a8b3)

**Comment by nortikin** [comment]
так-то matrix apply делает то же самое, но как опция не помшает.

**Comment by satabol** [comment]
> так-то matrix apply делает то же самое, но как опция не помшает.

Matrix Apply is post process. This socket work in process (as a parameter). You can not get this result after this node.
